manga-loader.user.js

```diff
@@ -1358,7 +1358,11 @@ var extractInfo = function(selector, mod, context) {
       case 'a':
         if(mod.type === 'index')
           return parseInt(elem.textContent);
-        return elem.href || elem.getAttribute('href');
+        var href = elem.href || elem.getAttribute('href');
+        if (!href.startsWith('//') && !href.startsWith('http') && !href.startsWith('#')) {
+          return window.location.href + "/../" + href;
```

Comment:
I might be missing something here, but is this code specific to Mangafox? Since this change is in the centrally used `extractInfo` function couldn't it affect a site that returns relative URLs.  My main concern is the `/../`.

Reply:
At least Firefox and Chrome normalize such sequences in URLs. See also https://tools.ietf.org/html/rfc3986#section-6.2.2.3
But you were right that it's not the best way to fix relative URL's, so I sat down for a while more and improved it.
Bato.to still doesn't work though.